index.html

```diff
@@ -1,143 +1,54 @@
-            alert(result.message);
-          } else {
-            const parse_format = 'DD.MM.YYYY HH:mm';
-            const display_format = 'dddd, MMM Do YYYY [at] HH:mm';
-            const m_date = moment(result.message.date, parse_format);
-            $('#event_date').text(m_date.format(display_format));
-            $('#event_location').text(result.message.location);
-            // update both text and href attribute
-            $('#payment_link').text(result.message.payment_link);
-            $('#payment_link').attr('href', result.message.payment_link);
+  <!--  css -->
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic|Anonymous+Pro:400,700,400italic,700italic|Merriweather:400,700,300">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <link rel="stylesheet" href="src/css/main.css?v=1.0.2">
 
-            $('.addeventatc .timezone').text('Europe/Amsterdam');
-            $('.addeventatc .title').text('Let\'s volleyball @' + result.message.location + ' on ' + m_date.format(display_format));
-            $('.addeventatc .description').text(result.message.date + ' ' + result.message.location);
-            $('.addeventatc .location').text(result.message.location);
+  <!--  js -->
+  <script src="https://cdn.jsdelivr.net/npm/vue@2.5.16/dist/vue.min.js"></script>
+  <script src="src/js/jquery-2.2.3.min.js"></script>
+  <script src="src/js/underscore-min.js"></script>
+  <script src="src/js/moment.min.js"></script>
+  <script type="text/javascript" src="https://addevent.com/libs/atc/1.6.1/atc.min.js" async defer></script>
+  <script src="src/js/vb.js"></script>
+  <script src="src/js/index.js"></script>
 
-            $('.addeventatc .start').text(m_date.format(parse_format));
-            $('.addeventatc .end').text(m_date.add(3, 'hours').format(parse_format));
-
-            $( "#players" ).html(template( {players: result.message.guests} ));
-          }
-        },
-        error: function(response){
-          alert('server error');
-        }
-      });
-    });
-  </script>
-  <body>
-    <div class="container-fluid">
-      <div class="row">
-        <div class="col-md-10 offset-md-1 col-sm-12">
-          <div class="row">
-              <nav class="navbar navbar-expand-lg col bg-light">
-                <span class="navbar-brand">Let's volleyball</span>
-                <ul class="navbar-nav mr-auto">
-                  <li class="nav-item active">
-                    <a class="nav-link" href="#">Primary event</a>
-                  </li>
-                  <li class="nav-item">
-                    <a class="nav-link" href="events.html">All events</a>
-                  </li>
-                  <li class="nav-item">
-                    <a class="nav-link" href="shame.html">Shame</a>
```

Comment:
This change breaks navigation (`Shame` is missing for example).

Reply:
readded